build.py

```diff
@@ -71,12 +71,12 @@
 #
 
 DEFAULT_TRITON_VERSION_MAP = {
-    "release_version": "2.56.0dev",
-    "triton_container_version": "25.03dev",
-    "upstream_container_version": "25.01",
+    "release_version": "2.55.0",
```

Comment:
Are we dropping `dev` suffix?

Reply:
We are not. This was a miss on my end. Updated.